samples/event-logger/src/components/InputBoard/Filter/Filter.js

```diff
@@ -0,0 +1,137 @@
+import Heading from '@enact/sandstone/Heading';
+import Picker from '@enact/sandstone/Picker';
+import SwitchItem from '@enact/sandstone/SwitchItem';
+import PropTypes from 'prop-types';
+import {Component} from 'react';
+import {connect} from 'react-redux';
+
+import {activateEvent, isSyntheticEventOn, setDelayMs, setEventCapturing} from '../../../actions/actions';
+import eventCategory from '../../../constants/eventCategory';
+
+import css from './Filter.module.less';
+
+class FilterBase extends Component {
+	static propTypes = {
+		onActivateEvent: PropTypes.func,
+		onIsSyntheticEventOn: PropTypes.func,
+		onSetDelayMs: PropTypes.func,
+		onSetEventCapturing: PropTypes.func
+	};
+
+	constructor (props) {
+		super(props);
+	}
+
+	handleEventCategory = (index) => {
+		const {onActivateEvent} = this.props;
+
+		return function ({selected}) {
+			onActivateEvent(index, selected);
+		};
+	};
+
+	handleTimerPicker = ({value}) => {
+		const timergroup = [3000, 5000, 10000];
+		this.props.onSetDelayMs(timergroup[value]);
+	};
+
+	handleEventCapturing = ({selected}) => {
+		this.props.onSetEventCapturing(selected);
+	};
+
+	handleSyntheticEventOn = ({selected}) => {
+		this.props.onIsSyntheticEventOn(selected);
+	};
+
+	render () {
+		const
+			timergroup = ['3 sec', '5 sec', '10 sec'],
+			eventItems = eventCategory.map((e, i) => {
+				const handler = this.handleEventCategory(i);
+				return (
+					<SwitchItem
```

Comment:
Let's make all `SwitchItem`s to have `size` prop "small".

Reply:
It's added.